apps/assistants/sync.py

```diff
@@ -592,8 +599,15 @@ def link_files_to_vector_store(
                 "static": {"max_chunk_size_tokens": chunk_size, "chunk_overlap_tokens": chunk_overlap},
             }
 
-        for chunk in chunk_list(file_ids, 500):
-            self.client.vector_stores.file_batches.create(
-                vector_store_id=vector_store_id, file_ids=chunk, chunking_strategy=chunking_strategy
+        try:
+            for chunk in chunk_list(file_ids, 500):
+                self.client.vector_stores.file_batches.create(
+                    vector_store_id=vector_store_id, file_ids=chunk, chunking_strategy=chunking_strategy
+                )
+            return vector_store_id
+        except Exception as e:
+            logger.exception(
```

Comment:
Maybe drop this to a warning?

Reply:
Sure. https://github.com/dimagi/open-chat-studio/pull/1571/commits/9992537797335f1990c35d78a1af45df57f3cb54